components/cosmetic_filters/resources/data/element_picker.html

```diff
@@ -85,26 +85,42 @@
     }
 
     #main-section:not(.desktop) {
-      transition: transform 0.3s ease;
-      transform: translateY(calc(100% - 28px));
+      width: 30px;
+      padding: 0px;
+      height: 30px;
+      left: 47%;
```

Comment:
is this `left: 47%` to center it? If so does setting `margin-inline: auto` work?